helm/promxy-app/templates/deployment.yaml

```diff
@@ -27,13 +27,42 @@ spec:
       containers:
       - name: {{ include "name" . }}
         image: "{{ .Values.Installation.V1.Registry.Domain }}/{{ .Values.promxy.image.name }}:{{ .Values.promxy.image.tag }}"
-        ports:
-        - containerPort: {{ .Values.promxy.port }}
         args:
+        - "--bind-addr=:{{ .Values.promxy.port }}"
         - "--config={{ .Values.promxy.mountPath }}/config.yaml"
         - "--web.enable-lifecycle"
+        # - "--web.external-url"
```

Comment:
Was this meant to have a value here?

Reply:
This is to not forget as we will need to set that when we create the ingress for it